lib/smart_proxy_container_gateway/database.rb

```diff
@@ -0,0 +1,24 @@
+require 'sequel'
+  
+module Proxy
+  module ContainerGateway
+    class Database
+
+      def initialize(sqlite_db_path:, timeout:)
+        @connection = Sequel.connect("sqlite://#{sqlite_db_path}", timeout: timeout)
```

Comment:
In my other PR I made it more generic: if you only present the URL and do the string interpolation in the plugin then you could in theory also use the same code for PostgreSQL.

Reply:
I think I'll leave making this generic to when we actually support multiple databases -- I realize I'll need special logic to determine if we need to connect via sqlite and checks for if the sqlite PRAGMAs should be run. Bit of overhead for something we don't really need yet.